cmd/neofs-lens/internal/object/get.go

```diff
@@ -0,0 +1,38 @@
+package object
+
+import (
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/engine"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/spf13/cobra"
+)
+
+var getObjectCMD = &cobra.Command{
+	Use:   "get",
+	Short: "Get object from NeoFS",
+	Long:  "Get object from NeoFS",
+	Args:  cobra.NoArgs,
+	Run:   getObject}
+
+func init() {
+	common.AddAddressFlag(getObjectCMD, &vAddress)
+	common.AddOutputFileFlag(getObjectCMD, &vOut)
+	common.AddConfigFileFlag(getObjectCMD, &vConfig)
+}
+func getObject(cmd *cobra.Command, _ []string) {
+	var addr oid.Address
+
+	err := addr.DecodeString(vAddress)
+	common.ExitOnErr(cmd, common.Errf("invalid address argument: %w", err))
+
+	storage := openEngine(cmd)
+	defer storage.Close()
+
+	data, err := engine.Get(storage, addr)
+	common.ExitOnErr(cmd, common.Errf("could not fetch object: %w", err))
+
+	common.PrintObjectHeader(cmd, *data)
+	file, err := data.Marshal()
```

Comment:
```suggestion
	data, err := data.Marshal()
```

well, that is the _data_ now (`objRaw` if you wish)